packages/remark-mdx/test/fixtures/import-export.js

```diff
@@ -0,0 +1,21 @@
+module.exports = [
+  {
+    description: 'Handles basic usecase of default export',
+    mdx: 'export default props => <article {...props} />'
+  },
+  {
+    description: 'Handles an import that is later exported',
+    mdx: ["import Layout from './Foo'", '', 'export default Layout'].join('\n')
+  },
+  {
+    description: 'Separates import from the default export',
+    mdx: [
+      "import Foo from './foo'",
+      'export default props => <article {...props} />'
+    ].join('\n')
+  },
+  {
+    description: 'Handles other types of exports',
+    mdx: 'export const metadata = { some: "stuff" }'
```

Comment:
Do we need to add a case here for exports split across multiple lines?

```
description: 'Handles other types of exports',
mdx: [
  'export const metadata = {'
  '  some: "stuff"'
  '}'
].join('\n')
```

Reply:
I can add in a test for that 👍 . This still won't handle empty lines yet but I think that will be in my second pass because it will require a bit more complexity to be able to group separate nodes together for a single, valid import/export.